index.html

```diff
@@ -5,19 +5,45 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-   <script src="https://kit.fontawesome.com/8c1870b6fa.js" crossorigin="anonymous"></script>   
+    <script
+      src="https://kit.fontawesome.com/8c1870b6fa.js"
+      crossorigin="anonymous"
+    ></script>
     <title>Fatima's Portfolio</title>
   </head>
   <body>
-   
     <!--header section starts-->
     <header>
       <div class="logo"><a href="#">Fatima</a></div>
       <nav>
         <div class="hamburger">
-          <svg width="750px" height="750px" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" fill="none">
-            <path stroke="#6070FF" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 8h22M5 16h22M5 24h22"/>
+          <svg
+            id="show-nav"
+            class="nav-menu-button"
+            width="750px"
+            height="750px"
+            viewBox="0 0 32 32"
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+          >
+            <path
+              stroke="#6070FF"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M5 8h22M5 16h22M5 24h22"
+            />
           </svg>
+          <!-- mobile-menue start -->
+          <div id="mobile-menu" class="mobile-menu--hide">
+            <i id="hide-nav" class="fa-solid fa-x mobile-menu-close-icon"></i>
+            <ul id="mobile-menu-links">
+              <li><a href="#my-portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
+          </div>
+          <!-- mobile-menue finish -->
```

Comment:
When the user clicks (or taps) any of the mobile menu options, the correct part of the page should be displayed.  However, that is not implemented yet in your design.  Kindly ensure that it takes you to the selected/clicked menu since that is part of the project requirements. :+1: